gov_uk_dashboards/lib/dap/get_dataframe_from_cds.py

```diff
@@ -0,0 +1,86 @@
+""" Returns a dataframe after connecting to CDS, otherwise uses a csv already saved in the file"""
+import json
+import pandas as pd
+import pyodbc
+import boto3
+from botocore.exceptions import ClientError, NoCredentialsError
+
+CONN_STRING = (
+    "Driver={SQL Server};"
+    "Server=DAP-SQLTEST\\CDS;"
+    "Database=Dashboards;"
+    "Trusted_Connection=yes;"
+)
+
+CONN_STRING_DAP = (
+    "Driver={/usr/lib/libmsodbcsql-18.so};"
+    "Server=10.154.41.171\\CDS,1444;"
+    "TrustServerCertificate=yes;"
+    "Database=Dashboards;"
+)
+
+
+def data_from_cds_if_connected(query: str, csv_path: str):
```

Comment:
I think we talked about refactoring this to make it more descriptive, with it moving into the package now might be a good time to make those changes. This function goes on to get the data from whichever source it can, so I think it'd be good to give this a more general name and then create helper functions for each way of getting the data